src/css/custom.css

```diff
@@ -58,7 +58,7 @@ table {
   background-color: #1a1f30;
 }
 
-@media screen and (min-width: 996px) and (max-width: 1200px) {
+@media screen and (min-width: 996px) and (max-width: 1239px) {
```

Comment:
Why 1239? seems like a weird number?

Reply:
Because 1239px is when the current navbar content breaks in to two lines. 😉

Reply:
front-end is wonderful and messy. 😅